Autopark/Autopark/Vehicle.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Autopark
+{
+    class Vehicle : IComparable
+    {
+        public VehicleType Type { get; set; }
+        public string ModelName { get; set; }
+        public string RegistrationNumber { get; set; }
+        public int Weight { get; set; }
+        public int ManufactureYear { get; set; }
+        public int Mileage { get; set; }
+        public Color CarColor { get; set; }
+        public double TankCapacity { get; set; }
+
+        public Vehicle()
+        {
+        }
+
+        public Vehicle(VehicleType type, string modelName, string registrationNumber, int weight, int manufactureYear, int mileage, Color carColor, double tankCapacity)
+        {
```

Comment:
If a method or constructor takes more than 2 parameters, then each parameter should be written on a new line